src/resources/UsageAnalytics/UsageAnalytics.ts

```diff
@@ -18,5 +20,6 @@ export default class UsageAnalytics extends Resource {
         this.exports = new Exports(api, serverlessApi);
         this.statistics = new Statistics(api, serverlessApi);
         this.administration = new Administration(api, serverlessApi);
+        this.snowflake = new Snowflake(api, serverlessApi);
```

Comment:
UT for this one?

Reply:
Is it already covered by https://github.com/coveo/platform-client/blob/master/src/resources/tests/PlatformResources.spec.ts#L202 ? Otherwise, I didn't found the test file of `UsageAnalytics.ts`, I'm not sure what to test?